libs/langchain/langchain/utilities/dalle_image_generator.py

```diff
@@ -21,6 +24,8 @@ class DallEAPIWrapper(BaseModel):
     """Size of image to generate"""
     separator: str = "\n"
     """Separator to use when multiple URLs are returned."""
+    model: str = "dall-e-3"
```

Comment:
should we remove the default to have it automatically use OpenAI's default (currently dalle 2), to not have it auto-upgrade people unexpectedly?
```suggestion
    model: Optional[str] = None
```

Reply:
Yeah the default should match the underlying API